entries/r/raisin.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Raisin": {
+    "domain": "raisin.com",
```

Let's work under the assumption that they do share an authentication backend.

```suggestion
    "domain": "raisin.com",
    "additional-domains": [
      "weltsparen.at",
      "weltsparen.de",
      "raisin.nl",
      "raisin.es",
      "raisin.fr",
      "raisin.ie",
      "raisin.co.uk"
    ],
```